tests/release_tests/test_launch/launch-agent-patch.yaml

```diff
@@ -0,0 +1,39 @@
+metadata:
```

Comment:
This looks like the content of the files, whats the intended way for testers to use these? We should document that

Reply:
The tester doesn't need to interact with the patch files at all, the test handles it. See `_create_config_files()`. I'll add a comment in the yamls explaining them.